tests/test_il_finance_authority.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from os.path import dirname, join
+
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.il_finance_authority import IlFinanceAuthoritySpider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "il_finance_authority.html"),
+    url="https://www.il-fa.com/public-access/board-documents/",
+)
+
+test_pdf_response = file_response(
+    join(dirname(__file__), "files", "il_finance_authority.pdf"),
+    url=("https://www.il-fa.com/"),
+    mode="rb",
+)
+
+spider = IlFinanceAuthoritySpider()
+
+freezer = freeze_time("2020-09-30")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+pdf_text = spider._parse_agenda_pdf(test_pdf_response)
+
+freezer.stop()
+
+
+def test_time():
+    time = spider._parse_start(pdf_text)
```

Comment:
This should work, but is there a reason for not testing against a Meeting item? Usually that's the way we've done it to make sure the whole process works

Reply:
To be honest, I'm not sure how to test the id since it  doesn't appear from parsed_items. Perhaps I'm missing the obvious from my novice coding skills, but I can't seem to think of way to get all the information to recreate//test the id. Since the process of yielding a a meeting object is very specific in order and 'tricky' for this spider.

Reply:
Gotcha, we aren't typically using the `meta` attribute on requests, but it seems like the right approach here and testing it can be tricky.

Here's an example of how we're doing that for the `chi_buildings` scraper where we're using it in a similar way. Let me know if that helps!

https://github.com/City-Bureau/city-scrapers/blob/c42620480f9184ca74885e2a3c6ef576faba3eb7/tests/test_chi_buildings.py#L22-L32